index.js

```diff
@@ -95,7 +105,10 @@ class SimpleKeyring extends EventEmitter {
   // For eth_decryptMessage:
   async decryptMessage(withAccount, encryptedData) {
     const wallet = this._getWalletForAccount(withAccount);
-    const privateKey = ethUtil.stripHexPrefix(wallet.privateKey);
+    let { privateKey } = wallet;
+    if (typeof privateKey === 'string') {
```

Comment:
this part scares me a bit. private key is inconsistently string or buffer on the wallet obj?

Reply:
yeah this was an oversight. It should only ever be a buffer. Will remove.

Reply:
I was thrown off by the use of `stripHexPrefix`, previously, but its not actually doing anything.